# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../qmail/qmail-ldap-use-ipsvd.diff
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff --git a/qmail-smtpd.sh b/qmail-smtpd.sh
index fc24d9f..400cb7d 100644
--- a/qmail-smtpd.sh
+++ b/qmail-smtpd.sh
@@ -28,8 +28,9 @@ fi
 
 exec \
 	envuidgid $QUSER \
-	tcpserver -v -URl $ME -x$QMAIL/control/qmail-smtpd.cdb \
-	    ${CONCURRENCY:+"-c$CONCURRENCY"} ${BACKLOG:+"-b$BACKLOG"} 0 smtp \
-	$PBSTOOL \
-	$QMAIL/bin/qmail-smtpd
+	tcpsvd -vvh -u $QUSER -l $ME -i ./peers \
+		${CONCURRENCY:+"-c$CONCURRENCY"} ${BACKLOG:+"-b$BACKLOG"} \
+		${IP:-0} 25 \
+		$PBSTOOL \
+		$QMAIL/bin/qmail-smtpd
 
